vignettes/communicate.Rmd

```diff
@@ -233,6 +233,16 @@ Then describe why the function was superseded, and what the recommended alternat
 
 The rest of the documentation can stay the same.
 
+In the body of the function add a call to `signal_stage()`:
```

Comment:
Should we wait until we've used this in real cases before documenting it prominently? Related to the comment about labelling signal-stage as experimental.